onnx2torch/node_converters/activations.py

```diff
@@ -75,17 +83,21 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
 
 @add_converter(operation_type='LogSoftmax', version=13)
 def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint: disable=unused-argument
+    dim = node.attributes.get('axis', -1)
```

Comment:
Take it back the way it was.

Reply:
This style not only my decision. We use this in concat, gather, flatten converters